projects/1.16/assets/mystical-world/mysticalworld/lang/zh_cn.json

```
@@ -1 +1,434 @@
+    "item.mysticalworld.sapphire": "蓝宝石",
+    "item.mysticalworld.sapphire_axe": "蓝宝石斧",
+    "item.mysticalworld.sapphire_boots": "蓝宝石靴子",
+    "item.mysticalworld.sapphire_chestplate": "蓝宝石胸甲",
+    "item.mysticalworld.sapphire_helmet": "蓝宝石头盔",
+    "item.mysticalworld.sapphire_hoe": "蓝宝石锄",
+    "item.mysticalworld.sapphire_knife": "蓝宝石刀",
+    "item.mysticalworld.sapphire_leggings": "蓝宝石护腿",
+    "item.mysticalworld.sapphire_pickaxe": "蓝宝石镐",
+    "item.mysticalworld.sapphire_shovel": "蓝宝石锹",
+    "item.mysticalworld.sapphire_spear": "蓝宝石矛",
+    "item.mysticalworld.sapphire_sword": "蓝宝石剑",
+    "item.mysticalworld.silk_cocoon": "蚕茧",
+    "item.mysticalworld.silk_thread": "丝线",
+    "item.mysticalworld.silkworm_egg": "蚕卵",
+    "item.mysticalworld.silkworm_spawn_egg": "蚕刷怪蛋",
+    "item.mysticalworld.silver_axe": "银斧",
+    "item.mysticalworld.silver_boots": "银靴子",
+    "item.mysticalworld.silver_chestplate": "银胸甲",
+    "item.mysticalworld.silver_dust": "银粉",
+    "item.mysticalworld.silver_fox_spawn_egg": "银狐刷怪蛋",
+    "item.mysticalworld.silver_helmet": "银头盔",
+    "item.mysticalworld.silver_hoe": "银锄",
+    "item.mysticalworld.silver_ingot": "银锭",
+    "item.mysticalworld.silver_knife": "银刀",
+    "item.mysticalworld.silver_leggings": "银护腿",
+    "item.mysticalworld.silver_nugget": "银粒",
+    "item.mysticalworld.silver_pickaxe": "银镐",
+    "item.mysticalworld.silver_shovel": "银锹",
+    "item.mysticalworld.silver_spear": "银矛",
+    "item.mysticalworld.silver_sword": "银剑",
+    "item.mysticalworld.sliced_carrot": "胡萝卜条",
+    "item.mysticalworld.spindle": "纺锤",
+    "item.mysticalworld.sprout_spawn_egg": "芽精刷怪蛋",
+    "item.mysticalworld.stewed_eggplant": "炖茄子",
+    "item.mysticalworld.stone_knife": "石刀",
+    "item.mysticalworld.stone_spear": "石矛",
+    "item.mysticalworld.stuffed_aubergine": "填馅茄子",
+    "item.mysticalworld.tannin_vial": "丹宁酸瓶",
+    "item.mysticalworld.tin_axe": "锡斧",
+    "item.mysticalworld.tin_boots": "锡靴子",
+    "item.mysticalworld.tin_chestplate": "锡胸甲",
+    "item.mysticalworld.tin_dust": "锡粉",
+    "item.mysticalworld.tin_helmet": "锡头盔",
+    "item.mysticalworld.tin_hoe": "锡锄",
+    "item.mysticalworld.tin_ingot": "锡锭",
+    "item.mysticalworld.tin_knife": "锡刀",
+    "item.mysticalworld.tin_leggings": "锡护腿",
+    "item.mysticalworld.tin_nugget": "锡粒",
+    "item.mysticalworld.tin_pickaxe": "锡镐",
+    "item.mysticalworld.tin_shovel": "锡锹",
+    "item.mysticalworld.tin_spear": "锡矛",
+    "item.mysticalworld.tin_sword": "锡剑",
+    "item.mysticalworld.vegetable_juice": "蔬菜汁",
+    "item.mysticalworld.venison": "鹿肉",
+    "item.mysticalworld.vinegar": "醋",
+    "item.mysticalworld.wasp_attractant": "黄蜂引诱剂",
+    "item.mysticalworld.wood_knife": "木刀",
+    "item.mysticalworld.wood_spear": "木矛",
+    "item.mysticalworld.young_pearl": "未熟珍珠",
+    "itemGroup.mysticalworld": "魔幻世界",
+    "message.dandelion_cordial": "你感到气色不错！",
+    "message.shoulder.occupied": "你的肩膀被其他事物所占！",

```

```suggestion
    "message.dandelion_cordial": "你感觉精力充沛！",
    "message.shoulder.occupied": "你的肩膀被其他事物所占！",
```
下面这句是什么时候出现的
是肩膀上已经停了只鹦鹉的时候吗（）